client/src/client-only-routes/show-unsubscribed.tsx

```diff
@@ -1,4 +1,5 @@
 import { Grid, Panel, Button } from '@freecodecamp/react-bootstrap';
+import { RouteComponentProps } from '@reach/router';
```

Comment:
a nitpick, you can use the keyword type to indicate that it's a type for readability.

```suggestion
import type { RouteComponentProps } from '@reach/router';
```